tiptapy/__init__.py

```diff
@@ -85,8 +85,10 @@ class Embed(BaseContainer):
     def inner_render(self, node) -> str:
         attrs = node['attrs']
         html = attrs.get('html', '')
-        if attrs.get('type') == 'video' and attrs.get('caption').strip():
-            html += f"<figcaption>{attrs['caption']}</figcaption>"
+        if attrs.get('type') == 'video':
```

Comment:
@gauravr  updated the code with changes. Also, I have updated the description of the PR please have a look into that too. 😄

Reply:
@Pradhvan tests?

Reply:
@gauravr I have added initial stub test cases for now. I will be creating a separate PR with refactored test cases to work on the issue mentioned with the caption. 
> As @shon suggested we need three tests here
-- with caption "Extraction or whatever"
-- with no caption key
-- optionally an empty caption

Reply:
Update ! @gauravr 

 As @shon suggested we need three tests here, I have refactored `test_tranform.py` to test all the three conditions.

- [x] with caption "Extraction or whatever"

- [x] with no caption key

- [x] optionally an empty caption

* All the media tags `["image", "featured image", "embed"]` now test for the above three caption condition. 

* Updated `featuredimage.html` to be the expected HTML response.